demisto_sdk/commands/content_graph/interface/neo4j/queries/import_export.py

```diff
@@ -33,8 +33,8 @@ def merge_duplicate_content_items(tx: Transaction) -> None:
     run_query(
         tx,
         """// Merges possible duplicate content item nodes after import
-MATCH (n:BaseContent{not_in_repository: true})
-MATCH (m:BaseContent{content_type: n.content_type})
+MATCH (n:BaseNode{not_in_repository: true})
+MATCH (m:BaseNode{content_type: n.content_type})
```

Comment:
```suggestion
MATCH (n:{ContentType.BASE_NODE}{{not_in_repository: true}})
MATCH (m:{ContentType.BASE_NODE}{{content_type: n.content_type}})
```
don't forget to add the f before the string